src/components/vr_cooperation/src/process_data_request.cc

```diff
@@ -45,8 +45,8 @@ using json_keys::kInfo;
 CREATE_LOGGERPTR_GLOBAL(logger_, "ProcessDataRequest")
 
 ProcessDataRequest::ProcessDataRequest(
-    VRModule* parent,
-    const vr_hmi_api::ServiceMessage& message)
+      VRModule* parent,
+      const vr_hmi_api::ServiceMessage& message)
   : BaseCommandRequest(parent, message) {
```

Comment:
Wrong indent of 3 lines now. You had wrong indent in 50 line: [Constructor_Initializer_Lists](https://google.github.io/styleguide/cppguide.html#Constructor_Initializer_Lists)